packages/http-client-java/generator/http-client-generator-core/src/main/java/com/microsoft/typespec/http/client/generator/core/template/PomTemplate.java

```diff
@@ -220,45 +211,33 @@ protected void writeStandAlonePlugins(XmlBlock pluginsBlock) {
             pluginBlock.tag("groupId", "org.apache.maven.plugins");
             pluginBlock.tag("artifactId", "maven-compiler-plugin");
             pluginBlock.tag("version", "3.13.0");
-            pluginBlock.block("configuration", configurationBlock -> {
-                configurationBlock.tag("release", JavaSettings.getInstance().isAzureV1() ? "11" : "17");
-            });
+            pluginBlock.block("configuration", configurationBlock -> configurationBlock.tag("release",
+                JavaSettings.getInstance().isAzureV1() ? "11" : "17"));
         });
 
         // maven-source-plugin
         pluginsBlock.block("plugin", pluginBlock -> {
             pluginBlock.tag("groupId", "org.apache.maven.plugins");
             pluginBlock.tag("artifactId", "maven-source-plugin");
             pluginBlock.tag("version", "3.3.1");
-            pluginBlock.block("executions", executionsBlock -> {
-                executionsBlock.block("execution", executionBlock -> {
-                    executionBlock.tag("id", "attach-sources");
-                    executionBlock.block("goals", goalsBlock -> {
-                        goalsBlock.tag("goal", "jar");
-                    });
-                });
-            });
+            pluginBlock.block("executions", executionsBlock -> executionsBlock.block("execution", executionBlock -> {
```

Comment:
It seems there are quite a few formatting changes like this one in this PR (please let me know if I’m mistaken). Could you share how these changes were made? I’m just trying to understand whether they are related to in-memory formatting, as I thought in-memory formatting only applies to generated SDK code rather than the codegen code itself.